src/_includes/banner.html

```diff
@@ -7,9 +7,7 @@
     https://github.com/flutter/website/issues/3897
   {% endcomment %}
   <div class="site-banner site-banner--default" role="alert">
-  Check out <a href="https://medium.com/flutter/whats-new-in-flutter-2-0-fe8e95ecc65">today's announcements</a>,
-  or join us for the
-  <a href="https://events.flutter.dev">10:00 am (IST) / 4:30 am (UTC) restream</a>
+  Flutter 2 is here. <a href="https://medium.com/flutter/whats-new-in-flutter-2-0-fe8e95ecc65">Learn more</a>
```

Comment:
I don't like overuse of exclamation points, but today I think we get one :) And how about an ellipsis at the end? For example:

```
  Flutter 2 is here! <a href="https://medium.com/flutter/whats-new-in-flutter-2-0-fe8e95ecc65">Learn more...</a>
```

Reply:
I'm with you on the `!`. As for the ellipsis... apparently [many people see ellipses as passive aggressive](https://workplace.stackexchange.com/questions/149145/is-the-use-of-ellipsis-dismissive-or-rude)